### PR TITLE
[Remove Blocking usages]LPS-189564 portal-kernel | WorkflowComparatorFactoryUtil

### PR DESCRIPTION
**Pull request by JanisZhang**
Resend. previous:https://github.com/brianchandotcom/liferay-portal/pull/137958

Based on https://github.com/brianchandotcom/liferay-portal/pull/137958#issuecomment-1639053873 , I moved the `EDIT_PORTLET_DISPLAY_CONTEXT` into module `portal-workflow-web` in https://github.com/liferay-core-infra/liferay-portal/commit/d7753a1cef8ac73d7bb12d724192e98d473c54dd

Thanks!

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:startup

**Comment by JanisZhang** [comment]
ci:test:workflow

File: modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/constants/WorkflowInstancePortletConstants.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.workflow.web.internal.constants;
+
+/**
+ * @author Janis Zhang
+ */
+public class WorkflowInstancePortletConstants {
```
Comment (dantewang) [inline modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/constants/WorkflowInstancePortletConstants.java:20]: @JanisZhang Could you rename this class to `WorkflowInstancePortletWebKeys`? A constant that is used as request attribute key should be put in `WebKeys` class; a lot of other modules also have both `*Constants` and `*WebKeys` classes at the same time.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by JanisZhang** [comment]
Hi @dantewang, I append one SF commit. Thanks! cc @catarineaguiar

**Review by dantewang — APPROVED** [review]
@JanisZhang Looks good to me, please merge your SF into your original commit to make the history clean, thanks!

**Comment by JanisZhang** [comment]
Force-pushed based on the comment. Thanks! cc @dantewang

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/5086

**Pull request closed without merging**